mindjunkies/forums/forms.py

```
@@ -1,13 +1,24 @@
-        fields = ["title", "content"]
+        fields = ["title", "content","module"]
+
+    def __init__(self, *args, **kwargs):
+        # Get the course object from kwargs (passed when initializing the form)
+        course_slug = kwargs.pop('course_slug', None)
+        super().__init__(*args, **kwargs)
+
+        # If course is provided, filter the available modules based on the course
+        if course_slug:
+            course = get_object_or_404(Course, slug=course_slug)
+            self.fields['module'].queryset = Module.objects.filter(course=course)    
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)